src/VisualStudio/VisualBasic/Impl/Options/AdvancedOptionPageControl.xaml

```diff
@@ -104,17 +104,23 @@
             <GroupBox x:Uid="InlineHintsGroupBox"
                       Header="{x:Static local:AdvancedOptionPageStrings.Option_Inline_Hints_experimental}">
```

Comment:
Didn't we remove (experimental) in a previous PR? Did we miss this?

Reply:
No.  This has remained experimental.  And i thnk it should stay that way for a little bitlonger until we can play around with this feature work.